### PR TITLE
Update UG and DG Week 11: Search and Project

### LINKED ISSUE #156: Week 11: Update UG and DG for Search feature

### PR DESCRIPTION
**Pull request by florentianayuwono**

### CODE REVIEW

**Review by Ugholaf — COMMENTED** [review]
LGTM, just the Ui.png need to be updated since the Ui has changed from my latest PR.

**Comment by florentianayuwono** [comment]
Yes true, working on it now!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -237,18 +237,14 @@ Step 3. The `search` command then updates _filtered_ person list to contains the
 
 The following sequence diagram shows how the search operation works:
 
-(insert sequence diagram here)
+![SearchSequenceDiagram](images/SearchSequenceDiagram.png)
 
 <div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `SearchCommand` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```
Comment (Eugene-Ong-W-X) [inline docs/DeveloperGuide.md]:
```suggestion
<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `SearchCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```

**Review by Eugene-Ong-W-X — COMMENTED** [review]
One small error, good job

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (florentianayuwono) [reply docs/DeveloperGuide.md]: Nice catch, thanks!!

**Pull request merged**